### PR TITLE
feat(openomni): add windowed Stakes primitive

### PR DESCRIPTION
**Pull request by INONONO66**
Closes #469

## Outcome

Adds the dormant deterministic windowed ledger-observed Stakes primitive and public `@openomni/openomni/ledger` driver. This leaf wires no completion admission, durable-ledger cutover, Jester/Voice runtime, allocation, or dispatch authority.

## Contract delivered

- Snapshot-first owner/window/action/state/Stakes schemas with private authority graphs and fresh shared primitive factories.
- Policy/window/input/reference identities, aggregate-before-round money, owner/window isolation, set-union novelty, theta `1000`.
- Per-host broker with non-serializable completion and Voice capabilities plus WeakMap provenance.
- Completion binding covers owner/action/window/sequence/version/basis/range/novelty plus WorkItem hash and contract revision.
- Voice binding covers the shared basis plus evaluation and an independently host-read authorization receipt.
- Clone/local recompute, foreign broker, cross-surface, stale issuance basis, and subject transplantation fail closed.
- Frozen non-authorizing assessment and denial/success receipts.
- Scenario-specific discriminated driver statuses: `ok` and `resultCode` cannot contradict each other.
- Zero production callers; `docs/implementation-status.md` records the primitive and seams as dormant.

## Reconcile-first baseline

At refreshed `origin/main` (`425166d01d1a1edb10174ea8822c4d37d0890d38`), no Stakes implementation or consumer existed; `work.complete.pre`, Jester, and Voice remained target-only. No protocol vocabulary or policy point is added.

## Exact head

- commit: `58064d13c2779d6f5c958828fada49ac2cf21841`
- tree: `dc31172c8cceb5afd9ac77592e853e9f95eb1548`

## Verification

- prescribed baseline: 48 pass, 0 fail, 265 assertions
- focused Stakes contract: 58 pass, 0 fail, 148 assertions
- every completion/Voice binding, authority snapshot, Voice authorization, surface literal, and authority-request source field discriminated
- exact window/input/reference/archive goldens plus non-empty known-fingerprint and tuple-field discrimination
- workspace typecheck: 11/11
- workspace build: 4/4
- root lint: 895 files clean
- conformance + self-test, dependency gate, dead-export ratchet + self-test: pass
- `git diff --check`: pass
- hosted checks: 11 successful, 1 intentional skip, 0 pending/failing
- PR state: `MERGEABLE` / `CLEAN`
- review threads: 13 total, 13 resolved

## Exact-head Manual QA

- happy jq conjunction: true
- hostile jq conjunction: true
- happy raw line SHA-256: `512a52de65824f80d04210cb63cca256e6953df45d08dea63fb0cda372c3e527`
- hostile raw line SHA-256: `46cf417529246ea56d7bbbf0928580e7909452fdc236d9aa964c7aca47befe3d`
- happy archived input: `sha256:1d9d26fe9b1b3d69d4efdabacce0cda7cbd4d4545326cb2886655666de9f78d6`
- happy seam reference: `sha256:39ef5ca5472ebc3f8f26f413e2d6127c47a4168560cec60ba84b318e45b6ea44`
- hostile archived input: `sha256:ca6235fdfb6b4296f181f33b2fec919fb4b620eaa507bedf3fdfdbd06007a789`
- hostile preserved reference: `sha256:6f6457e462c8012c5e166cdf26801199309e0a228bb8b40fe261bd7f83b56da8`
- invalid arguments: typed `invalid_arguments`, exit 1
- help: exact deterministic usage
- `apps/server` public import: policy v1, `ok: true`, boundary value 1000

## Local pre-existing test gaps

- Cross-package root `bun test --timeout 15000` stops at the pre-existing real-IPC ordering after all Stakes tests; that IPC test passes isolated in about 0.55 seconds. Hosted CI uses package shards.
- The unrelated macOS fixed-sleep TERM marker test at `packages/openomni/src/execution-runtime/tool/builtins/bash.test.ts:93` remains outside this diff. Hosted Linux Test is green.

## Review gates

- [x] Required hosted build/test/type/lint/security checks green
- [x] CodeRabbit and Cubic findings fixed or evidence-dispositioned
- [x] All 13 review threads resolved
- [x] Exact-head goal/code/QA reviews MERGE-SAFE
- [x] Claude Code CLI exact-head adversarial review MERGE-SAFE
context reviews MERGE-SAFE/- [x] Exact-head security/context reviews MERGE-SAFE/context reviews MERGE-SAFE
- [x] Final exact-head receipt posted

### CODE REVIEW

File: packages/openomni/test/ledger/stakes-seam-binding-cases.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  })),
+  mutation("workItemHash", (binding) => ({ ...binding, workItemHash: "wi_other" })),
+  mutation("contractRevision", (binding) => ({
+    ...binding,
+    contractRevision: binding.contractRevision + 1,
+  })),
+];
+
+const voiceMutations: readonly VoiceMutation[] = [
+  ...completionMutations.slice(0, 8).map(({ name, denial, apply }) => ({ name, denial, apply })),
```
[the remark this answers is not in the document]
Comment (INONONO66) [reply packages/openomni/test/ledger/stakes-seam-binding-cases.ts]: Fixed in `d15fa076`: the positional `.slice(0, 8)` is gone. Completion and Voice tables now both consume a named `sharedBindingMutations` catalog from `stakes-seam-shared-mutations.ts`; surface-specific cases remain explicit. Focused suite stays 57/0/146, typecheck and root lint pass, and both test modules remain below 250 lines.

File: packages/openomni/src/ledger/stakes-driver-scenarios.ts [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+export type StakesDriverScenario = "threshold-and-split" | "forged-local-value";
+export type StakesDriverReceipt = Readonly<{
+  version: "stakes-driver-v1";
+  mode: "scenario";
+  scenario: StakesDriverScenario;
+  ok: boolean;
+  resultCode: "threshold_and_split_verified" | "forged_local_value_denied";
+}> &
+  Readonly<Record<string, unknown>>;
```
[the remark this answers is not in the document]
Comment (INONONO66) [reply packages/openomni/src/ledger/stakes-driver-scenarios.ts]: Fixed in `11b01e6a`. `StakesDriverReceipt` now intersects a scenario-specific discriminated status union: each `ok: true` branch has only its passing code and each `ok: false` branch has an explicit failure code. Both builders derive `ok` and `resultCode` together through `stakesDriverStatus`; regression coverage asserts both failure mappings. Focused suite is now 58/0/148, typecheck/build/lint pass, and the status contract was split into its own 46-line module to keep every module below 250 lines.

File: packages/openomni/test/ledger/stakes-seam-shared-mutations.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  mutation("noveltyBasisDigest", (binding) => ({
+    ...binding,
+    noveltyBasisDigest: stakesDigest("other-novelty-basis"),
+  })),
+];
+
+function mutation<T>(
+  name: string,
+  apply: (binding: T) => unknown,
+  denial: "binding_mismatch" | "invalid_subject" = "binding_mismatch",
```
[the remark this answers is not in the document]
Comment (INONONO66) [reply packages/openomni/test/ledger/stakes-seam-shared-mutations.ts:52]: Fixed in `58064d13`: the generic `mutation()` factory is now exported from `stakes-seam-shared-mutations.ts` and imported by `stakes-seam-binding-cases.ts`; the duplicate implementation is deleted. Focused suite remains 58/0/148 and typecheck/lint pass.
